doc/user_doc/vic_installation/plugin_h5_vc_web.md

```diff
@@ -0,0 +1,32 @@
+# Install the HTML5 Plug-In on vCenter Server for Windows by Using a Web Server #
+
+If your vCenter Server instance runs on Windows, you can use a Web server to host the HTML5 vSphere Client plug-in for vSphere Integrated Containers Engine.
+
+**Prerequisites**
+
+- You are running a Web server that your vCenter Server instance can access.
+- Use a Windows system to run the script to install the plug-in on a vCenter Server that runs on Windows. If you used a Linux or Mac OS system to deploy the VCH, download and unpack the vSphere Integrated Containers Engine package on a Windows system. For example, download the package to the system on which vCenter Server is running.
+
+**Procedure**
+
+1. On the Windows system on which you have downloaded and unpacked vSphere Integrated Containers Engine, navigate to the folder that contains the `vic-machine` utility and open the `ui` folder.
+2. Upload the plug-in bundle to your Web server.
+    <pre><i>unpack_dir</i>\vic\ui\plugin-packages\com.vmware.vic-<i>version</i>.zip</pre>
+3. On the system on which the Web server is running, make sure that port 3443 is open for both inbound and outbound connections.
+3. On the `vic-machine` system, open the <code><i>vic_unpack_dir</i>\vic\ui\vCenterForWindows\configs</code> file in a text editor.
+4. Enter the IPv4 address or FQDN of the vCenter Server instance on which to install the plug-in.<pre>SET target_vcenter_ip=<i>vcenter_server_address</i></pre>
+5. Enter the path to the folder on your Web server that contains the <code>com.vmware.vic-<i>version</i>.zip</code> file. For example, if you saved the ZIP file in a folder named `vsphere-plugins`, set the following URL: <pre>SET vic_ui_host_url="<i>web_server_address</i>:3443/vsphere-plugins"</pre>
```

Comment:
Added "You are running a Web server that your vCenter Server instance can access over HTTPS" to the prereqs. Added `https` to the URLs.